index.html

```diff
@@ -11,7 +11,13 @@
     <header id="header">
         <img src="https://www.flaticon.com/premium-icon/icons/svg/502/502337.svg" alt="logo" id="header-img" width='40'
             height='40' />
-        <nav id='nav-bar'></nav>
+        <nav id='nav-bar'>
+            <ol>
```

Comment:
Instead of a <ol></ol> you should probably use a <ul></ul>. I will merge this anyway but you should fix this on the next PR.